system/HTTP/CURLRequest.php

```diff
@@ -452,9 +452,31 @@ public function send(string $method, string $url)
 		{
 			$output = substr($output, strpos($output, "\r\n\r\n") + 4);
 		}
+		
+		// Set the string we want to break our response from
+		$break_string = "\r\n\r\n";
```

Comment:
Please, change it to `$breakString = "\r\n\r\n";` then place it right after line 449 and replace all occurrences of `"\r\n\r\n"` with this variable.